anvio/variabilityops.py

```diff
@@ -780,7 +780,7 @@ def get_sample_ids_of_interest(self, sample_ids_of_interest_path=""):
            method.
         """
         # use class-wide attribute if no parameters is passed
-        if sample_ids_of_interest_path is "":
+        if sample_ids_of_interest_path == "":
```

Comment:
Using `""` as some secondary version of `None` really exhibits the awkwardness of my teenaged years.